spec/factories/defence_counsels.rb

```diff
@@ -12,4 +12,16 @@
       defence_counsel.attendance_days << FactoryBot.build(:attendance_day)
     end
   end
+
+  factory :realistic_defence_counsel, class: 'DefenceCounsel' do
+    title { Person::TITLES.sample }
+    firstName { Faker::Name.first_name }
+    middleName { Faker::Name.middle_name }
+    lastName { Faker::Name.last_name }
+    status { Faker::Demographic.race }
+    after(:build) do |defence_counsel|
```

Comment:
**not a blocker**: have you considered using `associations` instead
[cheatsheet](https://devhints.io/factory_bot#associations)

Reply:
This would be nicer, however it doesn't work for our cases as some of the relationships are one sided (primarily due to our lack of understanding on HMCTS' implementation)